README.md

```diff
@@ -109,6 +109,13 @@ docker run --rm -td -p 8080:8080 --env GITHUB_USERNAME=XXXXXXX --env GITHUB_APPI
 
 #### Option 2
 
+~~~bash
+cp .env.template .env
+docker-compose up -d gad gad-ngrok
```

Comment:
Should you link off to ngrok?  Explain which account type is required for this?  Which params are from where, etc?